cs/filesystem.texy

```diff
@@ -0,0 +1,97 @@
+
+Práce se souborovým systémem - Nette\Utils\FileSystem
+*****************************************************
+
+.[perex]
+[Nette\Utils\FileSystem |api:] je statická třída s užitečnými funkcemi pro práci se souborovým systémem.
+
+
+Všechny příklady předpokládají vytvořený alias:
+
+/--php
+use Nette\Utils\FileSystem;
```

Comment:
Třeba u prvního příkladu by to mohlo být. Pro nás je to samozřejmé, ale dovedu si představit, že si na tom někdo vyláme zuby.